### PR TITLE
Added disabled state to list management component

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Components/ListManagment/ListManagment.tsx [hunk 1]
```diff
@@ -100,13 +102,14 @@ export const ListManagment = <T,>(
...
                     <List
+                      disabled={props.disabled}
                       list={visibleList}
                       options={props.options}
                       onClick={props.onClick}
                       renderItem={props.renderItem}
                     />
 
-                    {limit < filteredList.length && (
+                    {limit < filteredList.length && !props.disabled && (
```
Comment (martinweingart) [inline src/Components/ListManagment/ListManagment.tsx:112]: Why this? The user should still be able to list all the participants

File: src/Components/ListManagment/List/List.tsx [hunk 2]
```diff
@@ -5,13 +5,15 @@ import { Divider } from '../../Divider/Divider';
 import { ListItem, ListManagmentMenuOption } from './ListItem/ListItem';
 
 export interface ListProps<T> {
+  disabled?: boolean;
```
Comment (martinweingart) [inline src/Components/ListManagment/List/List.tsx:8]: A `disabled` prop it's not very clear. Instead, add two props: `isCreateButtonDisabled` and `showItemMenu`

File: src/Components/ListManagment/ListManagment.tsx (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Components/ListManagment/ListManagment.tsx]: Removed

File: src/Components/ListManagment/List/List.tsx (hunk 2 above)
Comment (Yopo-Nicolas) [reply src/Components/ListManagment/List/List.tsx]: Changed

**Review by martinweingart — APPROVED** [review]
LGTM

**Pull request merged**